src/content-scripts/canvas.js

```diff
@@ -54,6 +57,31 @@ function toggleCanvasOff(){
   console.log('canvas removed!');
 };
 
+// // utlity function to save serialized canvas data
+// function saveCanvas(){
+
+// };
+
+function serializeOut = function() {
+  data = ctx.toDataURL();
+  localStorage.setItem('OurCanvas', data);
+};
+
+function serializeIn = function() {
+  return storage.getItem('OurCanvas');
+};
+
```

Comment:
getCopyCanvas could be split into two or more functions that follow this linear process:
(1) retrieve canvas data
(2) create new canvas element
(3) append this canvas to the body with proper styling and attributes
(4) draw new canvas element according to data retrieved